sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -26,7 +26,9 @@ pub mod client;
 mod invocation;
 pub use invocation::Invocation;
 
-/// Result type that uses a [`proto::status::Status`] type for error values.
+/// Result type that uses a [`Status`] type for error values.
+///
+/// [`Status`]: oak_abi::proto::google::rpc
```

Comment:
Should this be

```suggestion
/// [`Status`]: oak_abi::proto::google::rpc::Status
```

?

Reply:
Yes! Thanks. Done.